test/battle/move_animations/smack_down.c

```diff
@@ -0,0 +1,25 @@
+#include "global.h"
+#include "test/battle.h"
+
+SINGLE_BATTLE_TEST("Move Animation Test: Smack Down works when used 15 times in a row")
+{
+    gBattleTestRunnerState->forceMoveAnim = TRUE;
+    u16 j, nTurns = 15;
+
+    GIVEN {
+        PLAYER(SPECIES_WOBBUFFET);
+        OPPONENT(SPECIES_WOBBUFFET);
+    } WHEN {
+        for (j = 0; j < nTurns; j++)
+        {
+            TURN { MOVE(player, MOVE_SMACK_DOWN); MOVE(opponent, MOVE_HELPING_HAND); } // Helping Hand, so there's no anim on the opponent's side.
+        }
+    } SCENE {
+        for (j = 0; j < nTurns; j++)
+        {
+            ANIMATION(ANIM_TYPE_MOVE, MOVE_SMACK_DOWN, player);
+        }
+    } THEN {
+        gBattleTestRunnerState->forceMoveAnim = FALSE;
```

Comment:
And to complement the potential macro above, this could automatically be set to `FALSE` at the end of the test, so that you only need to add a single line per test that you want to force move animations.

Reply:
Hmm, not sure if that would be feasible. I can do `FORCE_MOVE_ANIM `, but without putting the whole `THEN` there

Reply:
Better than how it was earlier at least ^^;